tests/docker/config/resource-types.json

```diff
@@ -36,7 +36,8 @@
   "https://neuroshapes.org/NeuronCount",
   "https://neuroshapes.org/NeuronDensity",
   "https://neuroshapes.org/NeuronMorphology",
-  "https://neuroshapes.org/NISSLImageDataLayer",
+  "https://bbp.epfl.ch/ontologies/core/bmo/NeuronMorphologyFeatureAnnotation",
```

Comment:
You don't need to add a new type if they are not indexed separately from the `NeuronMorphology`